### ISSUE TITLE
New NRL names and abbreviations

### ISSUE THREAD

**Issue by afalenski** [SiLeBAT/mibi-portal-client#536]
As a user I would like to see the current NRL names and abbreviations in the MiBi-Portal / in the Einsendebogen sent by the MiBi-Portal

- Change lab abbreviation which is shown in the leftmost column of the data table and in the wizard
- Change lab name which is written next to "Labor" in the Einsendebogen
- Change lab abbreviation in the RegEx entry in the DB

NRL names and abbreviations as well as NRL RegEx sheet have been sent via email by AF 16.10.2023@13:00

**Comment by birgitlewicki** [comment]
Implemented and deployed to qa.
Please test

**Comment by afalenski** [comment]
TEST WILL BE CONTINUED ON ANOTHER DAY
- NRL abbreviations are correct in client.
- NRL abbreviations are correct in PDF name.
- NRL full names are correct in PDF lab name.

Bugs:
- NRL-Trichinella is not deleted. It is still shown as supposedly responsible lab when uploading pathogen "Genus Trichinella". Message "Dieser Erreger wird von keinem BfR-Labor untersucht" is not shown.
- When the pathogen "Escherichia coli O157:H7" is uploaded, NRL-VTEC is correctly recognised. However, the pathogen error "Erreger nicht erkannt" is shown. The helping function suggests "Escherichia coli O-157:H7". If this is selected, the lab is not recognised. 
Both errors might be solved by using a corrected RegEx table, sent by AF on 16.11.2023@19:14

**Comment by birgitlewicki** [comment]
Bug fix:
- the indicated regex is added to the NRL-VTEC selectors according to the corrected RegEx table sent by AF. Now the pathogen "Escherichia coli O157:H7" is recognised correctly when the suggested entry "Escherichia coli O-157:H7" is chosen from the dropdown box.
 
- The NRL-Trichinella is not handled here. As discussed the removel of this NRL will be done in a separate ticket.

**Comment by birgitlewicki** [comment]
Bug fix:
- the indicated regex is added to the NRL-VTEC selectors according to the corrected RegEx table sent by AF. Now the pathogen "Escherichia coli O157:H7" is recognised correctly when the suggested entry "Escherichia coli O-157:H7" is chosen from the dropdown box.
 
- The NRL-Trichinella is not handled here. As discussed the removel of this NRL will be done in a separate ticket.

**Comment by afalenski** [comment]
Yes, Escherichia coli O-157:H7 is recognised. When entering Escherichia coli O157:H7, the pathogen is not recognised and an error is shown (as it should be, because in the new AVV catalogue it is written with hyphen). The NRL, however, is recognised, which is not so bad.
Done.
